libs/metrics/top_level_metrics.py

```diff
@@ -332,16 +331,17 @@ def calculate_weekly_admissions_per_100k(
         ]
         first_cdc_admissions_index = cdc_admissions_per_100k.first_valid_index()
     else:
-        cdc_admissions_per_100k = pd.Series(dtype=float)
+        cdc_admissions_per_100k = EMPTY_TS
         first_cdc_admissions_index = None
 
     # Use HSA-level data for counties only.
-    if region.level == AggregationLevel.COUNTY:
+    if region.is_county():
         # Counties in the Northern Mariana Islands are not mapped to an HSA, so
         # they have no hsaPopulations. For these instances do not try and
-        # calculate a metric.
-        if hsa_population is None:
-            can_admissions_per_100k = pd.Series(dtype=float)
+        # calculate a metric. DC has state-level data, not county-level, so the HSA columns
+        # don't exist for it.
+        if hsa_population is None or region.fips == "11001":
+            can_admissions_per_100k = EMPTY_TS
```

Comment:
I'm confused.  Don't we want to hit the else block on line 348 so we use the HSS state-level data for DC?  So line 338 should be `if region.is_county() and region.fips != "11001":` like we did for covid patients ratio ([line 303](https://github.com/covid-projections/covid-data-model/pull/1298/files#diff-6d0e13950a596647213010e005f51de2cf5bd90346eab74741918df924fd35d4R303))?

Or am I missing something?

Reply:
🤔 Yeah, that seems to make more sense--and appears to be working locally. I'm creating a snapshot to verify now